pkg/csi/service/fcd/controller.go

```diff
@@ -118,11 +130,11 @@ func (c *controller) CreateVolume(
 	}
 
 	datastoreName := params[AttributeFirstClassDiskParentName]
+	zone := params[AttributeFirstClassDiskZone]
+	region := params[AttributeFirstClassDiskRegion]
```

Comment:
This is an area where I need to do more reading, and see how things are implemented in the sidecars, but I would not expect zone/region information to come in as attributes... I would have expected them to be in the `TopologyRequirement` within the [spec](https://github.com/container-storage-interface/spec/blob/v1.0.0/spec.md#createvolume)

Reply:
I will double check to see if this is already being sent over. If it is, I will delete that and pull from the appropriate place and update the documentation.

Reply:
i dumped the topology fields in the Create request…. and nothing is set. seems like a bug on the kubernetes side. until that is fixed, we need those labels.